pkg/pvcresizing/pvc_test.go

```diff
@@ -1,11 +1,14 @@
 package pvcresizing
 
 import (
+	"flag"
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
 )
 
+var _ = flag.Bool("update", false, "doing nothing")
```

Comment:
Added to avoid errors when updating the test files

Reply:
Done